lib/install/management/store_files.go

```diff
@@ -83,7 +88,7 @@ func (d *Dispatcher) deleteImages(conf *config.VirtualContainerHostConfigSpec) e
 			continue
 		}
 
-		if len(children) == 0 {
+		if len(children) == 0 || len(children) == 1 { // when image store path is given, we need to delete VIC dir too when it is empty.
```

Comment:
Do we still have the scenario len(children)==0?

Reply:
Good catch. Remove that condition.